StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1009ClosingParenthesisMustBeSpacedCorrectly.cs

```diff
@@ -78,26 +79,12 @@ private void HandleCloseParenToken(SyntaxTreeAnalysisContext context, SyntaxToke
                 return;
             }
 
-            bool precededBySpace;
-
-            bool followedBySpace;
-            bool lastInLine;
+            bool precededBySpace = token.IsFirstInLine() ? true : token.IsPrecededBySpace();
```

Comment:
You could also write this as `precededBySpace = token.IsFirstInLine() || token.IsPrecededBySpace();`